go.mod

```diff
@@ -5,7 +5,9 @@ go 1.12
 require (
 	github.com/gdamore/tcell/v2 v2.4.0
 	github.com/go-errors/errors v1.0.2
-	github.com/mattn/go-runewidth v0.0.10
+	github.com/mattn/go-runewidth v0.0.14
```

Comment:
I got tons of changes here and below from running `go mod tidy`. Do we want these here? Happy to drop them except for the ones that concern tcell.

Reply:
I decided to make this clearer by first bumping tcell to 2.6.0, which brings in all these changes, and then switch to my fork in a separate commit. Could you have a quick look @jesseduffield, I think this is ready to merge.